jac-uk/admin
# #1997 Exercise Delete Button - Main Overview Page

## What's included?
Closes #1997 

Note: in this PR a new exercise state `deleted` is introduced when an exercise is deleted.

Please let me know if you can think of a better idea to make the UI nicer.

<img width="1231" alt="Screenshot 2023-06-15 at 10 08 11" src="https://github.com/jac-uk/admin/assets/79906532/e91b6343-c308-4047-959c-38888b81bab0">


## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

1. Go to exercise list and select multiple exercises.
2. Press `Delete` button to delete selected exercises.

Ensure that you have the `canDeleteExercises` permission.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://github.com/jac-uk/admin/assets/79906532/7f0fd79a-c31b-4e17-bdd5-be8878514722

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
